contributing/development/core_and_modules/variant_class.rst

```diff
@@ -34,27 +32,30 @@ Basically, thanks to the Variant class, writing Godot itself was a much,
 much easier task, as it allows for highly dynamic constructs not common
 of C++ with little effort. Become a friend of Variant today.
 
+.. note::
+
+    Unlike Objects, Variants **cannot** be ``null`` and must always store a
```

Comment:
But this is not entirely true, `Variant` can't be typed and `null`, but can be `null` (which counts as own type). You can totally use untyped `Variant` variable as nullable `int` for example.

Reply:
I've reworded it accordingly, let me know if it's clear enough now.